pages/docs/config.mdx

```diff
@@ -1,56 +1,41 @@
 
-## Code Config
-
-To use Grouparoo to import and export your data, you will be configuring our main objects: Apps, Sources, Destinations, Properties, etc. You can [learn more about Grouparoo's product concepts here](/docs/getting-started/product-concepts). You will be using the Grouparoo CLI's `generate`, `validate`, and `apply` commands to ensure that the application can read from your Sources and write to your Destinations.
-
-Configuration files are either JSON or Javscript which allow you the flexibility to manipulate data or load configuration from external sources:
-
-```js
-// file: config/apps/data_warehouse.js
+<Button className="mb-4" variant="outline-primary">
+  Get Started with Code Config
+</Button>
 
-exports.default = async function GetDataWarehouseConfig() {
-  const { databasePassword } = await fetch(`/api/company/secrets`);
+<Alert variant="primary">
+  <p className="mb-0">
+    <strong>Note:</strong> Grouparoo's Enterprise Edition enables you to
+    configure Grouparoo through the UI.{" "}
+    <a href="/docs/config/ui-config">Learn more about UI Config here</a>.
+  </p>
+</Alert>
 
-  return [
-    {
-      name: "Data Warehouse",
-      class: "app",
-      id: "data_warehouse",
-      type: "postgres",
-      options: {
-        host: "127.0.0.1",
-        port: "5432",
-        username: "person",
-        password: databasePassword,
-        database: "data_warehouse",
-      },
-    },
-  ];
-};
-```
+## Code Config Guides
 
-You can learn more about the [configuration files here](/docs/deployment/code-config).
+Code Config is the recommended way to install Grouparoo. Here are some guides to help you through that process:
 
-<Button variant="outline-primary" href="/docs/cli/config">
-  Learn about the Generate Command
-</Button>
+- [Intro to Code Config](/docs/config/code-config)
+- [Adding Apps](/docs/config/apps/community)
+- [Adding Sources](/docs/config/sources/community)
+- [Adding Properties to Sources](/docs/config/properties/community)
+- [Adding Destinations](/docs/config/destinations/community)
+- [Creating Groups](/docs/config/groups/community)
 
-### Configuration Guides
+## UI Config (Enterprise) Guides
 
-Explore our guides and examples to start using Grouparoo:
+If you have [purchased a license to Grouparoo's Enterprise Edition](/meet), you can configure your application through the UI. Here are some guides to help you through that process:
 
-- [Installing Grouparoo](/docs/deployment)
-- [Setting up Grouparoo the first time](/docs/config/first-time-setup)
-- [Adding Apps](/docs/config/apps)
-- [Adding Sources](/docs/config/sources)
-- [Adding Destinations](/docs/config/destinations)
-- [Defining a Profile Property Rule](/docs/config/properties)
-- [Creating Groups](/docs/config/groups)
+- [Intro to UI Config](/docs/config/ui-config)
+- [Adding Apps](/docs/config/apps/enterprise)
```

Comment:
Per your "Discrepancy in Steps" note, I wonder if we add some notes here about Running and Deploying before using the UI to configure.